### PR TITLE
feat(sessions): Added duration to summaries and sorting

### PR DESCRIPTION
**Pull request by mitsuhiko**
This adds the ability to change the rollup for the summaries and sorting.

### CODE REVIEW

File: src/sentry/api/endpoints/organization_releases.py [hunk 1]
```diff
@@ -30,7 +30,13 @@
 from sentry.utils.cache import cache
 
 
-ERR_INVALID_STATS_PERIOD = "Invalid stats_period. Valid choices are '', '24h', and '14d'"
+SUMMARY_STATS_PERIOD = ("24h", "48h", "7d", "14d")
+HEALTH_STATS_PERIOD = ("", "24h", "14h")
```
Comment (matejminar) [inline src/sentry/api/endpoints/organization_releases.py]:
```suggestion
HEALTH_STATS_PERIOD = ("", "24h", "14d")
```

File: src/sentry/api/endpoints/organization_releases.py [hunk 2]
```diff
@@ -30,7 +30,13 @@
 from sentry.utils.cache import cache
 
 
-ERR_INVALID_STATS_PERIOD = "Invalid stats_period. Valid choices are '', '24h', and '14d'"
+SUMMARY_STATS_PERIOD = ("24h", "48h", "7d", "14d")
```
Comment (markstory) [inline src/sentry/api/endpoints/organization_releases.py]: Could this be pulled from the other definition in snuba/sessions?

File: src/sentry/snuba/sessions.py [hunk 3]
```diff
@@ -34,17 +34,23 @@ def get_changed_project_release_model_adoptions(project_ids):
...
-        "sessions_1h": (["sessions"], timedelta(hours=1)),
-        "sessions_24h": (["sessions"], timedelta(days=1)),
+    if stats_period is None:
+        stats_period = "24h"
+
+    _, stats_start, _ = get_rollup_starts_and_buckets(stats_period)
+
+    orderby = {
+        "crash_free_sessions": [["divide", ["sessions_crashed", "sessions"]]],
+        "crash_free_users": [["divide", ["users_crashed", "users"]]],
```
Comment (markstory) [inline src/sentry/snuba/sessions.py]: Does snuba handle sessions/users being 0 gracefully here?

**Review by markstory — APPROVED** [review]
Looks good to me other than the bit of duplication.

**Review by matejminar — APPROVED** [review]

File: src/sentry/snuba/sessions.py (hunk 3 above)
Comment (mitsuhiko) [reply src/sentry/snuba/sessions.py]: Turns into `inf`.

File: src/sentry/api/endpoints/organization_releases.py (hunk 2 above)
Comment (mitsuhiko) [reply src/sentry/api/endpoints/organization_releases.py]: I will want to refactor this a bit going forward. There is some duplication all around.

**Pull request merged**